apertium-kaa.kaa.lexc

```diff
@@ -4027,7 +4027,6 @@ wáde:wáde N1 ; ! ""
 wákil:wákil N1 ; ! ""
 wálayat:wálayat N1 ; ! ""
 wásiyyat:wásiyyat N1 ; ! ""
-waqıt:waq%{y%}t N1 ; ! ""
```

Comment:
Hey,

`%{y%}` means that in some forms the vowel can disappear (say, in poetry), as it should be the case in Qaraqalpaq too: `waqıt`, but `(onıń) waqtı`. Han be written like that?

If both forms -- `waqıtı` and `waqtı` exist, you should add both `waqıt` and `waq%{y%}` into the lexicon and mark the one which should only be analysed (not generated) as `Dir/LR`.

Reply:
no, the vowel doesn't disappear.

Reply:
If it never disappears, then we just need the `waqıt` entry.

Reply:
we have waqıt